packages/sdk-cli/src/models/HostConnections.ts

```diff
@@ -12,9 +15,37 @@ export class HostConnection {
     public host: RemoteHost,
   ) {}
 
+  static getDumpStreamTap() {
+    const dumpFileHandle = fs.openSync(process.env.FITBIT_DEVBRIDGE_DUMP_PATH as string, 'w');
+    const now = () => new Date().getTime();
+    const epoch = now();
+
+    function writeChunk(prefix: string) {
+      return (chunk: any) => fs.writeSync(
+        dumpFileHandle,
+        `[${prefix}][${now() - epoch}] ${JSON.stringify(chunk, undefined, 2)}\n`,
+      );
+    }
+
+    const transforms = {
+      preSerializeTransform: new StreamTap(writeChunk('send')),
+      postDeserializeTransform: new StreamTap(writeChunk('recv')),
+    };
+
+    stream.finished(transforms.postDeserializeTransform, () => fs.closeSync(dumpFileHandle));
+
+    return transforms;
+  }
+
   static async connect(hostID: string) {
     const ws = await developerRelay.connect(hostID);
-    return new this(ws, await RemoteHost.connect(ws));
+    return new this(
+      ws,
+      await RemoteHost.connect(
+        ws,
+        process.env.FITBIT_DEVBRIDGE_DUMP_PATH ? this.getDumpStreamTap() : undefined,
```

Comment:
seems like `getDumpStreamTap` is only used here, yet we process `process.env.FITBIT_DEVBRIDGE_DUMP_PATH` twice.

Maybe we should have something like 

`
await RemoteHost.connect(ws,this.getDumpStreamTap())
`

and have `process.env.FITBIT_DEVBRIDGE_DUMP_PATH` there, only once.